.github/workflows/test.yaml

```diff
@@ -0,0 +1,15 @@
+name: Test
+
+on: [ push ]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.6.0
+      - uses: awalsh128/cache-apt-pkgs-action@v1.3.0
+        with:
+          packages: lld
+      - uses: dtolnay/rust-toolchain@1.72.0
+      - run: cargo test --all-features
```

Comment:
Maybe we should run [`Swatinem/rust-cache`](https://github.com/Swatinem/rust-cache) just before running the tests?

Reply:
Let's do it in a separate PR so that we already have a minimal working CI for now